.travis.yml

```diff
@@ -2,10 +2,16 @@ language: php
 
 php:
  - 7.1
+ - 7.2
+ - nightly
+
+matrix:
+  allow_failures:
+    - php: nightly
 
 before_script:
  - travis_retry composer self-update
- - travis_retry composer install --no-interaction --prefer-source --dev
+ - travis_retry composer update --no-interaction --prefer-source
```

Comment:
What's your reasoning to change this? It could lead to false negatives, if a package violates semver.

Reply:
- The ```dev``` option is deprecated, and it will be the option by default.
- Update the dependencies is for the different PHP version tests in Travis build.

Reply:
Makes sense, thanks.